.swiftlint.yml

```diff
@@ -1,5 +1,6 @@
 disabled_rules:
 - trailing_whitespace
+- unneeded_break_in_switch
```

Comment:
It's better not to disable this rule for all source code. 
Instead, you can disable it for the next line of code only like so:

`// swiftlint:disable:next unneeded_break_in_switch`

More examples, you can see in this article https://milanpanchal24.medium.com/how-to-disable-swiftlint-rules-40dd3d7ba232